spec/lib/support_spec.rb

```diff
@@ -0,0 +1,31 @@
+Rails.application.load_tasks
+
+RSpec.describe "Support tasks" do
+  describe ":find_user" do
+    subject(:task) { Rake.application["support:find_user"] }
+
+    context "when there are no users" do
+      it "outputs the user doesn't exist" do
+        expect { task.execute({ email: "foo@example.gov.uk" }) }.to output("User 'foo@example.gov.uk' does not exist\n").to_stdout
+      end
+    end
+
+    context "when a user with email 'foo@example.gov.uk' exists" do
```

Comment:
Nitpick: I personally don't like specs with mutually exclusive `context`s like this.  Rather than

```ruby
describe "something to test" do
  context "when X is false" do
    ...
  end

  context "when X is true" do
    setup { ... }

    ...
  end
end
```

I'd usually remove the `context` around the `when X is false` case:

```ruby
describe "something to test" do
  ...

  context "when X is true" do
    before { ... }

    ...
  end
end
```

Though if both cases have some `before` I'd be more inclined to leave it.

But we don't have a standard for this, and other developers do it differently, so leave this if you want.

Reply:
Thinking about it a bit more, do we even need the test for when there are no users? I'm not sure we gain anything worthwhile from it. If you agree, we could get rid of that test and the whole 'falsey' context block

Reply:
Yeah, we already cover the case where the email address isn't found in other tests.  We can remove this.

Reply:
Removed in 2558f32